### ISSUE TITLE
x blurred some of your photos

### ISSUE THREAD

**Issue by malenki** [mapillary/mapillary_issues#901]
Now and then I receive notification emails that someone blurred some of
my images.
I am fine with that – but I would like to know which image(s) exactly
were (un)blurred. A link to it/them would be useful.

**Comment by AllenGraetz** [comment]
malenki, I've found that at least with some of these no blurring occurred.   I'm not sure why their system generates them when no change occured.  Maybe they're not checking to confirm there are dirty fields - aka fields that have been modified - when the form is submitted?  So if no blur was made, it still kicks off the notification process?

Getting these email notifications made me realize a history of image changes, at least a readable one, is very useful.  Ideally there would be a process for rolling back changes.

**Comment by pbb72** [comment]
I think also, especially when someone want to *un*blur my photo, Mapillary should ask for confirmation from me. The whole idea of blurring is to protect the privacy of people, but if anyone can unblur a photo without problems, there is no protection of the privacy...

**Comment by pbb72** [comment]
Could it be that some of the notification emails are not sent after someone (un)blurring my photo, but instead after someone adding/removing traffic signs from my photo? The photos I get these notifications from are often quite random, and not from people living nearby. Now looking at the traffic signs game, I see it is these people that are supposedly blurring my photos.

Of course, this could also just be coincidence.

**Comment by malenki** [comment]
Peter schrieb
am Mon, 13 Apr 2015 12:54:00 -0700:

> I think also, especially when someone want to *un*blur my photo,
> Mapillary should ask for confirmation from me. The whole idea of
> blurring is to protect the privacy of people, but if anyone can
> unblur a photo without problems, there is no protection of the
> privacy...

I happen to see a lot of blurred photos where the privacy of a houses
windows or of some traffic signs is protected. I can imagine that you'd
get tired of these messages in short time.

**Comment by jesolem** [comment]
@pbb72 just a comment: ALL edits, including un-blurs are moderated and need approval before they happen.

**Comment by pbb72** [comment]
@jesolem yeah, but not approval by the person who uploaded the photo, right?

**Comment by AllenGraetz** [comment]
The first problem I see with the idea of the photo uploader having veto
power on a blue edit is just that.  They may not want a needed blur edit..
The process should not depend on their approval.

Also, such a process would rely on the uploader still being active.  Most
users are only occasionally active.   The process would stall waiting for
their approval.  In the meantime, someone's privacy would be violated.


~Allen

**Comment by jesolem** [comment]
Yup. Mapillary (or in the future approved moderators) decides.

**Comment by pbb72** [comment]
Ok, I can see the problem with uploaders not wanting to accept required changes. But the confirmation process is quite slow (often taking several days) with very limited information. (Not showing if any change requests are pending.) This needs to change.

Back to the original issue (I think I've hijacked the thread): my email blur notifications do show which photos were (un)blurred, with a link to them. Though they don't say what exactly has been changed.
